### ISSUE TITLE
Incorrect Time for Import Conversation from Video

### ISSUE THREAD

**Issue by ichimura-a** [jkhartshorne/KidTalkIssues#36]
After I put in a date and time for when the imported audio was recorded, the time in the scrapbook was early by 4 hours.

iPhone 7 / Safari / iOS 13.6

For example, I had set the time of the imported audios for 9:59 am, but the scrapbook indicates it to have been recorded at 5:59 am.

![image](https://user-images.githubusercontent.com/68654587/90152909-54ed8680-dd56-11ea-95ae-e79a5e807ccf.png)

**Comment by jkhartshorne** [comment]
BTW We want the 'hour' tag turned off anyway (there's in the punchlist). Should be date only. But this could of course affect dates as well, if it's near midnight.

**Comment by ichimura-a** [comment]
@jkhartshorne If I input a time before 4 am, then the scrapbook will display the date of the previous day. Not sure if this would happen if the hour tag is removed.

**Comment by lorenachipstudio** [comment]
@jkhartshorne @ichimura-a We'll take a look at this- thanks!

**Comment by alexchippendale** [comment]
@ichimura-a There is an issue with iOS 13, iPadOS and some older versions on iOS 11/12 that causes the timezone to get shifted when selecting a date in web apps. This is a problem with those operating systems and we've been able to recreate it on an iPhone iOS 13. We're working on a fix from our side that will correct for it on those devices.

**Comment by alexchippendale** [comment]
@ichimura-a @jkhartshorne This is now fixed on staging. We actually found that this is a bug built into most mobile web browsers and added special handling to make sure the the time is converted correctly for your timezone (it had been defaulting to UTC). 

We fixed the underlying issue with time conversion and also removed the time tag from the scrapbook tiles. Note that that last part is just for the scrapbook tiles, we are also working on removing time selection altogether from imports in a separate issue (see https://github.com/alexchippendale/KidTalkRemainingTasks/issues/11). 

This fix solves the reported time conversion error "Incorrect Time for Import Conversation from Video". That means you shouldn't run into any problems of a shifting date if the recording is made close to midnight.
